### PR TITLE
Episode 3: improving image descriptions

### PR DESCRIPTION
**Pull request by vinisalazar**
- Improve image description for 5 figures in _episodes/03-matplotlib.md as suggested by @ivastar in #781

### CODE REVIEW

File: _episodes/03-matplotlib.md [hunk 1]
```diff
@@ -228,7 +228,7 @@ formats, including SVG, PDF, and JPEG.
 > > matplotlib.pyplot.show()
 > > ~~~
 > > {: .language-python}
-> ![Plot with step lines](../fig/inflammation-01-line-styles.svg)
+> ![Three line graphs, with step lines connecting the points, showing the daily average, maximum and minimum inflammation over a 40-day period.](../fig/inflammation-01-line-styles.svg)
```
Comment (ldko) [inline _episodes/03-matplotlib.md:231]: To keep this line under 100 characters, would you mind splitting the text on to a second line?

**Review by ldko — COMMENTED** [review]
Thanks for adding in the proposed alt text, @vinisalazar . I think it looks good. One comment included.

File: _episodes/03-matplotlib.md (hunk 1 above)
Comment (vinisalazar) [reply _episodes/03-matplotlib.md]: Thanks for pointing that out. Fixed by cf41831.

**Review by maxim-belkin — APPROVED** [review]
LGTM! 👍🏻

**Review by ldko — APPROVED** [review]
Looks good @vinisalazar !

**Pull request merged**